https://issues.redhat.com/browse/ACM-9851--adding entry to MCE/Observ… (#6201)
open-cluster-management/rhacm-docs: clusters/release_notes/whats_new.adoc

* You can now create a hosted cluster with the KubeVirt platform by using the console. For more information, see xref:../hosted_control_planes/create_cluster_kubevirt.adoc#hosted-create-kubevirt-console[Creating a hosted cluster by using the console].

* You can now configure additional networks, request a guaranteed CPU access for Virtual Machines (VMs), and manage scheduling of KubeVirt VMs for node pools. For more information, see xref:../hosted_control_planes/managing_nodepools_kubevirt.adoc#managing-nodepools-hosted-cluster-kubevirt[Configuring additional networks, guaranteed CPUs, and VM scheduling for node pools].

[#acm-integration]
== {product-title-short} integration

> If you have {product-title-short} installed and Observability enabled, you can now view your hosted control planes cluster capacity estimate and resource utilizations.

Reviewer: you can now view your hosted control planes cluster capacity estimate and resource utilizations by existing hosted control planes.

Author: changed

> If you have {product-title-short} installed and Observability enabled, you can use Grafana dashboards to view your hosted control planes cluster capacity estimate, and existing hosted control planes resource utilizations. See more at xref:../install_upgrade/acm_integration.adoc#acm-integration[{product-title-short} integration].

Reviewer 2: ```suggestion
If you enable Observability after you install {product-title-short}, you can use Grafana dashboards to view your hosted control planes cluster capacity estimate, and existing hosted control planes resource utilizations. See more at xref:../install_upgrade/acm_integration.adoc#acm-integration[{product-title-short} integration].
```
